client/utilities/benefitsThreshold.ts

```diff
@@ -0,0 +1,45 @@
+import type { CurrencyIso } from './currencyIso';
+
+export const benefitsThresholdsByCountryGroup: Record<
```

Comment:
Would it make sense to combine the benefit thresholds and contribution amounts into a single object so the config is in a single place? (Both have monthly and annual amounts for the same list of countries.)

Reply:
Referring to David's previous comment, this is probably best informed by any discussion around storing these values in a central location.

Reply:
I thought about that - but given it's still somewhat TBD it's probably better not to combine them before we know what we want to extract to a central location.